### PR TITLE
DL-1470 Migrated to Play 2.5.19 and bootstrap-play-25

### PR DESCRIPTION
**Pull request by CEPoole**
# DL-1470 Play 2.5.19 Security fix

**Maintenance**

Updated microservice to Play 2.5.19 due to security fix. As a result of deployment issues, this also required us to migrate to bootstrap-play-25.

## Checklist

 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [x]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/uk/gov/hmrc/residencenilratebandcalculator/views/rnrbHelpers/inputRadioGroup.scala.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
+@*
```
Comment (Ufuoma-Daniel) [inline app/uk/gov/hmrc/residencenilratebandcalculator/views/rnrbHelpers/inputRadioGroup.scala.html]: How come you moved from the govuk template to a custom radio group?

File: conf/application.conf [hunk 2]
```diff
@@ -20,20 +20,33 @@ appName="residence-nil-rate-band-calculator-frontend"
...
-# ~~~~
-# Set to the object handler for migrated 2.3 services
-play.http.requestHandler = "play.api.http.GlobalSettingsHttpRequestHandler"
+# Custom error handler
+play.http.errorHandler = "uk.gov.hmrc.residencenilratebandcalculator.utils.RnrbErrorHandler"
 
+cookie.encryption.key = "gvBoGdgzqG1AarzF1LY0zQ=="
+queryParameter.encryption.key = "gvBoGdgzqG1AarzF1LY0zQ=="
+sso.encryption.key = "gvBoGdgzqG1AarzF1LY0zQ=="
+play.crypto.secret="changethistosomethingsecret"
```
Comment (Ufuoma-Daniel) [inline conf/application.conf]: What is this secret being set to/being overridden by?

File: app/uk/gov/hmrc/residencenilratebandcalculator/views/rnrbHelpers/inputRadioGroup.scala.html (hunk 1 above)
Comment (CEPoole) [reply app/uk/gov/hmrc/residencenilratebandcalculator/views/rnrbHelpers/inputRadioGroup.scala.html]: It's not in bootstrap anymore - it's the exact same one that it used to be

File: conf/application.conf (hunk 2 above)
Comment (CEPoole) [reply conf/application.conf]: app-config-common

File: app/uk/gov/hmrc/residencenilratebandcalculator/views/rnrbHelpers/inputRadioGroup.scala.html (hunk 1 above)
Comment (jmountain-hmrc) [reply app/uk/gov/hmrc/residencenilratebandcalculator/views/rnrbHelpers/inputRadioGroup.scala.html]: It was renamed. You can get it from: `@import uk.gov.hmrc.play.views.html.helpers.input_radio_group`

**Pull request merged**